src/models/values.rs

```diff
@@ -67,6 +74,20 @@ macro_rules! decimal_scalar {
                     Ok(None)
                 }
             }
+
+            pub fn serialize_to_float<S>(&self, s: S) -> std::result::Result<S::Ok, S::Error>
```

Comment:
so I think you can't use the rust_decimal serde features directly? Why wrap?

Reply:
I left this here in case it was useful; when defining a struct (instead of using the json! macro), you can override the serializer with this. But we can remove this since it's unused.

Reply:
Yeah. Let’s remove unused code

Reply:
It's done